### PR TITLE
app-i18n/transifex-client: add version 0.13.12

### PR DESCRIPTION
**Pull request by robert7k**
```
$ diff -u transifex-client-0.13.9.ebuild transifex-client-0.13.12.ebuild 
--- transifex-client-0.13.9.ebuild      2020-05-13 22:42:46.529883365 +0200
+++ transifex-client-0.13.12.ebuild     2020-10-01 19:32:46.961720869 +0200
@@ -14,14 +14,21 @@
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64 x86"
+KEYWORDS="~amd64 ~x86"
 IUSE="test"
 RESTRICT="!test? ( test )"
 
 BDEPEND="test? ( dev-python/mock[${PYTHON_USEDEP}] )"
-RDEPEND="<dev-python/python-slugify-2.0.0[${PYTHON_USEDEP}]
-       >=dev-python/requests-2.19.1[${PYTHON_USEDEP}]
+RDEPEND="dev-python/GitPython[${PYTHON_USEDEP}]
+       <dev-python/python-slugify-5.0.0[${PYTHON_USEDEP}]
+       dev-python/requests[${PYTHON_USEDEP}]
        <dev-python/six-2.0.0[${PYTHON_USEDEP}]
-       >=dev-python/urllib3-1.24.2[${PYTHON_USEDEP}]"
+       dev-python/urllib3[${PYTHON_USEDEP}]"
 
 distutils_enable_tests setup.py
+
+src_prepare() {
+       eapply_user
+       sed -i -e 's:test_fetch_timestamp_from_git_tree:_&:' \
+               tests/test_utils.py || die
+}
```

### CODE REVIEW

File: app-i18n/transifex-client/transifex-client-0.13.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="test? ( dev-python/mock[${PYTHON_USEDEP}] )"
+RDEPEND="dev-python/GitPython[${PYTHON_USEDEP}]
+	<dev-python/python-slugify-5.0.0[${PYTHON_USEDEP}]
```
Comment (juippis) [inline app-i18n/transifex-client/transifex-client-0.13.11.ebuild:23]: Are you absolutely sure about these `<` deps? They make cleaning older versions pain.

File: app-i18n/transifex-client/transifex-client-0.13.11.ebuild [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	dev-python/requests[${PYTHON_USEDEP}]
+	<dev-python/six-2.0.0[${PYTHON_USEDEP}]
+	dev-python/urllib3[${PYTHON_USEDEP}]"
+
+distutils_enable_tests setup.py
```
Comment (juippis) [inline app-i18n/transifex-client/transifex-client-0.13.11.ebuild:28]:
```
Ran 103 tests in 0.215s

FAILED (failures=1)
Test failed: <unittest.runner.TextTestResult run=103 errors=0 failures=1>
error: Test failed: <unittest.runner.TextTestResult run=103 errors=0 failures=1>
 * ERROR: app-i18n/transifex-client-0.13.11::testworld failed (test phase):
 *   Tests fail with python3.6
```

Looks good but could you check whether this works for you? You can disable it this way:
https://dev.gentoo.org/~mgorny/python-guide/test.html#skipping-problematic-tests

File: app-i18n/transifex-client/transifex-client-0.13.11.ebuild (hunk 1 above)
Comment (robert7k) [reply app-i18n/transifex-client/transifex-client-0.13.11.ebuild:23]: They are explicitly mentioned by upstream, see https://github.com/transifex/transifex-client/blob/master/requirements.txt

**Comment by robert7k** [comment]
Updated this PR to add version 0.13.12 instead of 0.13.11.

File: app-i18n/transifex-client/transifex-client-0.13.11.ebuild (hunk 2 above)
Comment (robert7k) [reply app-i18n/transifex-client/transifex-client-0.13.11.ebuild]: Thanks for the hint. I excluded the failing test.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**